xml/ha_hawk2_manage_i.xml

```diff
@@ -0,0 +1,365 @@
+     resource to <literal>maintenance mode</literal> first. <remark>taroth 2015-12-04: krig - how
+      does that work with hawk2? does it work already? I searched in the Operations column for a
+      resource but could not find anything there...</remark>
+    </para>
+   </note>
+   <para> When creating a resource with &hawk;, you can set its initial state with the
+     <literal>target-role</literal> meta attribute. If you set its value to
+     <literal>stopped</literal>, the resource does not start automatically after being created. </para>
+   <procedure xml:id="pro.hawk2.rsc.start">
+    <title>Starting A New Resource</title>
+    <step>
+     <para>Log in to &hawk;2:</para>
+     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    </step>
+    <step>
+     <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para> Select the resource to start and in its <guimenu>Operations</guimenu> column click the
+       <guimenu>Start</guimenu> icon. To continue, confirm the message that appears. </para>
+     <para>As soon as the resource has started, &hawk; changes the resource's
+       <guimenu>Status</guimenu> to green and shows on which node it is running. </para>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.conf.hawk2.manage.cleanup">
+   <title>Cleaning Up Resources</title>
+   <para> A resource will be automatically restarted if it fails, but each failure increases the
+    resource's failcount. </para>
+   <para> If a <literal>migration-threshold</literal> has been set for the resource, the node will
+    no longer run the resource when the number of failures reaches the migration threshold. </para>
+   <para> A resource's failcount can either be reset automatically (by setting a
+     <literal>failure-timeout</literal> option for the resource) or you can reset it manually as
+    described below. </para>
+   <procedure xml:id="pro.hawk2.clean">
+    <title>Cleaning Up A Resource</title>
+    <step>
+     <para>Log in to &hawk;2:</para>
+     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    </step>
+    <step>
+     <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para> Go to the resource to clean up. In the <guimenu>Operations</guimenu> column click the
+      arrow down button and select <guimenu>Cleanup</guimenu>. To continue, confirm the message that
+      appears. </para>
+     <para>
+      <remark>taroth 2015-12-07: krig, is the following still true?</remark>This executes the
+      command <command>crm resource cleanup</command> for the specified resource on the specified
+      node. </para>
```

Comment:
Not quite, clean up cleans the resource on all nodes, not the specified node. But this was also the case in Hawk 1...